.github/workflows/assemble.yml

```diff
@@ -8,33 +8,22 @@ jobs:
     strategy:
       matrix:
         java: [ 21, 23 ]
-        os: [ubuntu-latest, windows-latest, macos-13, ubuntu-24.04-arm]
+        os: [ubuntu-latest, windows-latest, ubuntu-24.04-arm]
     steps:
       - uses: actions/checkout@v4
       - name: Set up JDK ${{ matrix.java }}
         uses: actions/setup-java@v4
         with:
           java-version: ${{ matrix.java }}
           distribution: temurin
-      - name: Setup docker (missing on MacOS)
-        id: setup_docker
-        if: runner.os == 'macos'
-        continue-on-error: true
-        run: |
-          brew install docker colima coreutils
-          gtimeout 15m colima start
-        shell: bash
-      - name: Run Gradle (assemble)
-        if: runner.os == 'macos' && steps.setup_docker.outcome != 'success'
-        run: |
-          # Report success even if previous step failed (Docker on MacOS runner is very unstable)
-          exit 0;
+        # GitHub Actions still not supporting arm64 dind yet
```

Comment:
It appears we have no path forward for `assemble` coverage on MacOS. That raises the question of why are we removing it now when it will still work for the next 5 months?

Reply:
I can keep the PR open in draft mode until then.
In any way until github actions update their M2 hardware to M3 or above it is hard to get dind run on arm64 macOS.

Thanks.